src/lib389/lib389/utils.py

```diff
@@ -309,20 +309,34 @@ def selinux_label_file(path, label):
             return
         raise ValueError(f'Cannot change file context for {path} because it is defined in SELinux policy. Please choose another path.')
     if label:
-        try:
-            log.debug(f"Setting label {label} in SELinux file context {path}.")
-            result = subprocess.run(["semanage", "fcontext", "-a", "-t", label, path],
-                                     stdout=subprocess.PIPE,
-                                     stderr=subprocess.PIPE)
-            args = ' '.join(ensure_list_str(result.args))
-            stdout = ensure_str(result.stdout)
-            stderr = ensure_str(result.stderr)
-            if result.returncode != 0:
-                log.error(f"ERROR CMD: {args} ; STDOUT: {stdout} ; STDERR: {stderr}")
+        rc = 0
```

Comment:
While you are here, could you please fix the docstring typo I've just noticed (wrong type label):

```
    :param label: The label to set (None to unset a label)
    :type path: str
```